### PR TITLE
Add link to phabricator to README.md

### PR DESCRIPTION
**Pull request by micgro42**
Since we are not using GitHub's issue functionality for this repository, we should provide users with an alternative way.

### CODE REVIEW

**Comment by wiese** [comment]
Thanks! Let's add https://docs.npmjs.com/files/package.json#bugs, too then.

**Comment by micgro42** [comment]
> Thanks! Let's add [docs.npmjs.com/files/package.json#bugs](https://docs.npmjs.com/files/package.json#bugs), too then.

Good idea, I adjusted my commit.

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
 	"version": "3.0.1",
 	"description": "WebdriverIO plugin for testing a Wikibase repo.",
 	"homepage": "https://github.com/wmde/wdio-wikibase",
+	"bugs": "https://phabricator.wikimedia.org/",
```
Comment (wiese) [inline package.json:6]: Sorry to drag this out but I always expected this to be a deep link. Arguably, `https://phabricator.wikimedia.org/tag/wikidata/` may not be an ideal tag to begin with and it certainly does not only contain topics related to this sub-component but a link to the home page does not seem to be in the best interest of the user in my eyes.

```suggestion
	"bugs": "https://phabricator.wikimedia.org/tag/wikidata/",
```

Comment (micgro42) [reply package.json:6]: Yeah, I considered that, but feared that seeing ~2000 "incoming" issues might be a bit disheartening? Maybe https://phabricator.wikimedia.org/tag/browser-tests/ would be better? At the very least, I should probably add that to the READEME.md link

Comment (wiese) [reply package.json:6]: Hmm. I don't know but seeing it was done with intent makes it ok with me (for lack of a truly good option). It's an improvement over the status quo and we don't plan to have it engraved in stone.

**Review by wiese — APPROVED** [review]
Thank you!

**Pull request merged**